pkg/abi/abi.go

```diff
@@ -736,7 +737,7 @@ func (e *Entry) SolidityDefCtx(ctx context.Context) (string, []string, error) {
 				if i > 0 {
 					buff.WriteString(", ")
 				}
-				s, childStructs, err := p.SolidityDefCtx(ctx, isFunction)
+				s, childStructs, err := p.SolidityDefCtx(ctx, true, false)
```

Comment:
I moved away from that, because it's not accurate here... even when `isFunction` is true, the value to this should be `false` - I will rename `isFunction` to `isFunctionInputs`

Reply:
second guessing myself here - checking on qualifiers for return

Reply:
ok - yeah, modifiers (`memory`) valid on returns. So code is correct. happy to change for readability.

Reply:
Happy to change readability - changed the assurance of these values to comments

Reply:
closing as covered by enum request below